src/imio/directory/core/contents/entity/export.py

```diff
@@ -0,0 +1,211 @@
+# # -*- coding: utf-8 -*-
+
+from collective.taxonomy.interfaces import ITaxonomy
+from imio.smartweb.common.utils import translate_vocabulary_term
+from plone import api
+from Products.Five.browser import BrowserView
+from zope.component import getSiteManager
+
+import io
+import json
+import pandas
+
+
+ordered_signifiant_columns = [
+    "title",
+    "title_fr",
+    "title_nl",
+    "title_de",
+    "title_en",
+    "subtitle",
+    "subtitle_fr",
+    "subtitle_nl",
+    "subtitle_de",
+    "subtitle_en",
+    "description_fr",
+    "description_nl",
+    "description_de",
+    "description_en",
+    "street",
+    "number",
+    "zipcode",
+    "city",
+    "country",
+    "country.title",
+    "complement",
+    "is_geolocated",
+    "geolocation.latitude",
+    "geolocation.longitude",
+    "mails",
+    "phones",
+    "urls",
+    "private_mails",
+    "private_phones",
+    "private_urls",
+    "private_note",
```

Comment:
Are we sure the rights to `ModifyPortalContent` will always be enough to see private informations ?